sabnzbd/newswrapper.py

```diff
@@ -139,6 +139,10 @@ def con(sock, host, port, sslenabled, write_fds, nntp):
                 except WantReadError:
                     select.select([sock], [], [], 1.0)
 
+            # Log SSL/TLS info
+            logging.info("%s@%s: Connected using %s (%s)",
```

Comment:
how costly is this to do over and over? maybe only do it once per server/per nzb? as logging == disk io.. which hurts sab performance all around